.github/workflows/tp.integration.tests.yml

```diff
@@ -92,14 +128,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
       - name: Cache Alpine Docker image
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

Comment:
Does v3 mean v3client?

Reply:
Nope, it means using [github.com/actions/cache/commits/v3](https://github.com/actions/cache/commits/v3) instead of [github.com/actions/cache/commits/v2](https://github.com/actions/cache/commits/v2).